const_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
```

Comment:
yep, not needed.  I just like having the labels since we also have a `nowindows` that matches against everything else.